app/assets/stylesheets/active_admin/components/_comments.scss

```diff
@@ -1,41 +1,25 @@
-// -------------------------------------- Admin Notes
-.comments {
+.active_admin_comment {
+  margin-bottom: 25px;
+  max-width: 45em;
+}
 
-  .active_admin_comment {
-    clear: both;
-    margin-top: 10px;
-    margin-bottom: 40px;
-    max-width: 700px;
+.active_admin_comment_meta {
+  color: $text-muted-color;
+  margin-bottom: 5px;
+}
 
-    .active_admin_comment_meta {
-      width: 130px;
-      float: left;
-      overflow: hidden;
-      font-size: 0.9em;
-      color: lighten($primary-color, 10%);
-      .active_admin_comment_author {
-        font-size: 1.2em;
-        font-weight: bold;
-        margin: 0;
-        color: $primary-color;
-      }
-    }
-    .active_admin_comment_body {
-      margin-left: 150px;
-    }
-  }
-  form.active_admin_comment {
-    margin: 0;
-    padding: 0;
-    margin-left: 150px;
+.active_admin_comment_author {
+  color: $primary-color;
+  font-weight: $font-weight-bold;
+  margin: 0;
+}
+
+.active_admin_comment fieldset.inputs {
```

Comment:
Merge rule `.active_admin_comment fieldset.inputs` with rule on line 1<br>Avoid qualifying class selectors with an element.<br>Selector `active_admin_comment` should be written in lowercase with hyphens